Analyzer/include/Config.h

```diff
@@ -231,6 +231,22 @@ class Config
             };
             registerModules(tr, std::move(modulesList));
         }
+        else if (analyzer=="AnalyzeLepTrigger") {
+            const std::vector<std::string> modulesList = {
+                "PartialUnBlinding",
+                "PrepNTupleVars",
+                "Muon",
+                "Electron",
+                "Photon",
+                "Jet",
+                "BJet",
+                "CommonVariables",
+                "Baseline",
+                "BTagCorrector",
+                "ScaleFactors"
+            };
+            registerModules(tr, std::move(modulesList));
+        }
```

Comment:
This this series of modules unique? If it is then this is fine if not then you can add it to the if statement.

Reply:
I think it is unique... What do you mean by "add it to the if statement".

Reply:
I mean add an `|| analyzer=="AnalyzeLepTrigger"` to the if statement corresponding to same series of modules, so we don't list the same series of modules more than once, but if it is unique then don't worry about it.

Reply:
Ah ok. Yeah, I believe it is technically unique.